docs/topics/KotlinBasic.md

```
@@ -0,0 +1,317 @@
+val c = String("abc")
+if(a==b) // 값 비교 - 참 // String의 equals 메소드를 호출. 그 문자열 클래스에서 생성(or할당)된 문자열들을 실질적으로 대소문자 같은지 비교.
+if(a===b) // 객체 참조값 비교 - 참 // 할당 - 상수영역 HashCode를 비교
+if(a===b) // 객체 참조값 비교 - 거짓 //생성 HashCode를 비교
+// Heap 메모리에 상수값 영역이 있다. 할당하면 "abc"는 상수영역을 참조
+// 생성하면 상수영역이 아닌 메모리영역을 참조
+
+var s = "abc" + 1 // s = "abc1"
+
+var s = "" // 공백도 객체다.
+for (a .. z) {
+	s += char // 문자열을 이런식으로 concat하면...
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)